Tools-Override/crossgen.sh

```diff
@@ -2,7 +2,9 @@
 
 # Restores crossgen and runs it on all tools components.
 
-__CoreClrVersion=1.1.0
+__CoreClrVersion=2.0.0-preview1-25204-02
```

Comment:
Technically I can pluck this version out of the string resources of libcoreclr.so. If `crossgen` is included with the shared framework though (which I think is the plan?) then it won't be necessary.